lib/resque/worker.rb

```diff
@@ -503,17 +503,15 @@ def heartbeat!(time = Time.now)
     end
 
     def start_heartbeat
-      heartbeat!
```

Comment:
I don't remember there being one, and I wrote it. 🙈  Pretty sure I would've written a failing test if there was.